manifest/spark-operator-rbac.yaml

```diff
@@ -34,7 +34,10 @@ rules:
   resources: ["pods"]
   verbs: ["*"]
 - apiGroups: [""]
-  resources: ["services", "configmaps", "secrets"]
+  resources: ["configmaps"]
```

Comment:
Why this change?

Reply:
The lock object is a ConfigMap, the leader election code needed some additional perms to work.